src/components/select-control/index.js

```
@@ -75,11 +75,13 @@ export default function SelectControl({
 	return (
 		!isEmpty(options) && (
 			<BaseControl
+				__nextHasNoMarginBottom
```

Biome

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)